### PR TITLE
add details about sample target

### PR DESCRIPTION
**Pull request by sbidari**
add more details re. submitting samples from a marginal/joint distribution with an example data table.

### CODE REVIEW

File: model-output/README.md [hunk 1]
```diff
@@ -217,8 +217,18 @@ Teams must provide the following 23 quantiles:
 
 #### sample output
 
-When the predictions are samples, values in the `output_type_id` column are indexes for the samples. 
-*More details to be added here*
+When the predictions are samples, values in the `output_type_id` column are indexes for the samples. The `output_type_id` is used to indicate the dependence across multiple task id variables when samples come from a joint predictive distribution. For example, samples from a joint predictive distribution across `horizon`, will share `output_type_id` for predictions for different horizons within a same `location` as below:
```
Comment (dylanhmorris) [inline model-output/README.md:220]:
```suggestion
When the predictions are samples, values in the `output_type_id` column are indexes for the samples. The `output_type_id` is used to indicate the dependence across multiple task id variables when samples come from a joint predictive distribution. For example, samples from a joint predictive distribution across `horizon`s for a given `location`, will share `output_type_id` for predictions for different horizons within a same `location` as below:
```

File: model-output/README.md [hunk 2]
```diff
@@ -217,8 +217,18 @@ Teams must provide the following 23 quantiles:
...
+| origin_date|horizon| location | output_type| output_type_id | value |
+|:---------- |:-----:|:-----:| :-------- | :------------ | :---- |
+| 2024-10-15 | -1    |  MA | sample | s0 | - |
+| 2024-10-15 |  0      |  MA | sample | s0 | - |
+| 2024-10-15 | 1       |  MA | sample | s0 | - |
+| 2024-10-15 | -1      |  MA | sample | s1 | - |
+| 2024-10-15 |  0      |  MA | sample | s1 | - |
+| 2024-10-15 |  1      |  MA | sample | s1 | - |
+
+Here, `output_type_id = s0` specifies that the predictions for horizons -1, 0, and 1 are part of the same joint distribution. More details on sample output can be found in the [hubverse documentation of sample output type](https://hubverse.io/en/latest/user-guide/sample-output-type.html).
```
Comment (dylanhmorris) [inline model-output/README.md:231]: For extra clarity, give an example of a second location, so people know how to indicate that MA trajectories are _not_ joint with, e.g., `NH` trajectories, but are joint across horizons for each location? And maybe also give an example of a submission of trajectories that are joint across both locations and horizons?

**Review by dylanhmorris — CHANGES_REQUESTED** [review]
Looks very good, thanks @sbidari! A few tweaks/suggestions.

File: hub-config/tasks.json [hunk 3]
```diff
@@ -132,12 +132,14 @@
                         "sample": {
                             "output_type_id_params": {
                                 "is_required": false,
-                                "type": "integer",
+                                "type": "character",
+                                "max_length": 15,
                                 "min_samples_per_task": 100,
-                                "max_samples_per_task": 100
+                                "max_samples_per_task": 100,
```
[the remark this answers is not in the document]
Comment (sbidari) [reply hub-config/tasks.json:138]: We could maybe request submission in `parquet` format then? Variant nowcasting hub is doing this

[the remark this answers is not in the document]
Comment (dylanhmorris) [reply hub-config/tasks.json:138]: I think we probably want to stick with `csv` for alignment with FluSight. We can do some rough calculations on what file size might look like with more samples allowed.

[the remark this answers is not in the document]
Comment (sbidari) [reply hub-config/tasks.json:138]: Okay, I can do this and run it by you

[the remark this answers is not in the document]
Comment (dylanhmorris) [reply hub-config/tasks.json:138]: Sounds good, thanks!

File: model-output/README.md (hunk 2 above)
Comment (sbidari) [reply model-output/README.md]: added another example table in https://github.com/CDCgov/covid19-forecast-hub/pull/8/commits/d66668cc241e2a6dfd4583af93c5dfb3584f0b8d

**Review by dylanhmorris — APPROVED** [review]
LGTM!

**Pull request merged**